### PR TITLE
Semidiscrete ot

### PR DESCRIPTION
**Pull request by BDoignies**
Added a new metric : semidiscrete optimal transport distance (to uniform distribution) in 2D.

### CODE REVIEW

**Review by dcoeurjo — APPROVED** [review]
That's great! thanks

File: setup.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 UTK_CMAKE_ARGS = [
     f"-DUTK_LOG=ON",
     f"-DUTK_USE_OPENMP=ON",
-    f"-DUTK_USE_FFTW=ON", 
+    f"-DUTK_USE_FFTW=OFF",
```
Comment (dcoeurjo) [inline setup.py:23]: disabling FFTW from python binding?

**Review by dcoeurjo — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (BDoignies) [reply setup.py:23]: Indeed, that is a mistake... On my computer, the conda environement I use do not like fftw for a reason I still don't know...

**Comment by dcoeurjo** [comment]
Et 👌

**Pull request merged**